bin/jsonschema_suite

```diff
@@ -52,33 +52,7 @@ REMOTES = {
 }
 REMOTES_DIR = os.path.join(ROOT_DIR, "remotes")
 
-TESTSUITE_SCHEMA = {
-    "$schema": "http://json-schema.org/draft-03/schema#",
-    "type": "array",
-    "items": {
-        "type": "object",
-        "properties": {
-            "description": {"type": "string", "required": True},
-            "schema": {"required": True},
-            "tests": {
-                "type": "array",
-                "items": {
-                    "type": "object",
-                    "properties": {
-                        "description": {"type": "string", "required": True},
-                        "data": {"required": True},
-                        "valid": {"type": "boolean", "required": True}
-                    },
-                    "additionalProperties": False
-                },
-                "minItems": 1
-            }
-        },
-        "additionalProperties": False,
-        "minItems": 1
-    }
-}
-
+TESTSUITE_SCHEMA = json.load(open("test-schema.json"))
```

Comment:
Also would be good to use a path relative to `__file__` rather than relative to cwd.